test/test_torch.py

```diff
@@ -8100,6 +8131,11 @@ def test_tensor_set(self):
         self.assertEqual(t1.size(), size)
         self.assertEqual(t1.stride(), stride)
 
+        t1 = torch.tensor([True, True], dtype=torch.bool)
```

Comment:
what about calling torch.tensor([True, True]) without also passing dtype = bool?  Is that coming in a later patch?

Reply:
it will come in the later patch